openmc/tallies.py

```diff
@@ -587,8 +672,20 @@ def get_tally_xml(self):
 
         return element
 
-
     def find_filter(self, filter_type):
+        """Return a filter in the tally that matches a specified type
```

Comment:
It doesn't look like `None` is ever returned; rather a `ValueError` is raised if no matching `Filter` is found. I'll add that in a "Raises" section (unless you feel that `None` should be returned rather than raising a ValueError).